source/ed/ed_persistor.cpp

```diff
@@ -130,7 +130,8 @@ void EdPersistor::insert_admins(const ed::Georef& data){
     for(const auto& itm : data.admins){
         if(itm.second->is_used){
             this->lotus.insert({std::to_string(itm.second->id), itm.second->name, itm.second->insee,
-                               itm.second->level, this->to_geographic_point(itm.second->coord),"admin:" + itm.second->insee});
+                               itm.second->level, this->to_geographic_point(itm.second->coord),
```

Comment:
This is the geopal part that calls persist(), that calls insert_admins().
So the code is not so consistently stored, but to me it's needed.